meeteval/wer/__main__.py

```diff
@@ -95,7 +95,7 @@ def _load(path: Path):
 
 def _load_reference(reference: 'Path | list[Path]'):
     """Loads a reference transcription file. Currently only STM supported"""
```

Comment:
`Currently only STM supported` <- That is no longer valid

Reply:
Done